src/CoreBundle/MapperLoader/MapperLoader.php

```diff
@@ -22,9 +22,6 @@ class MapperLoader implements MapperLoaderInterface
      */
     private $container;
 
-    /**
-     * @param ContainerInterface $container
-     */
     public function __construct(ContainerInterface $container)
```

Comment:
This is now a ServiceLocator (of Symfony, not the one from Chameleon). So I would suggest to reflect that in the type and the name. So rather use `ServiceLocator $locator`

Reply:
I don't really understand the method \Symfony\Component\DependencyInjection\Compiler\ServiceLocatorTagPass::register() which should set this parameter. (What is $id in the lower part of the method?)

The return type there is simply "Reference"?

But I used \Symfony\Component\DependencyInjection\ServiceLocator now as type.